west.yml

```diff
@@ -154,7 +154,7 @@ manifest:
       groups:
         - hal
     - name: libmetal
-      revision: 2f586b4f1276fb075ee145421bdf6cbe5403aa41
+      revision: pull/17/head
```

Comment:
@carlocaione 
Please, could you also update the associated open-amp module to be aligned between the libmetal and openamp release version.
The related openamp PR is https://github.com/zephyrproject-rtos/open-amp/pull/13